src/test/java/com/rackspace/ceres/app/services/MetadataServiceTest.java

```diff
@@ -306,6 +317,18 @@ void getMetricNames() {
     assertThat(results).containsExactlyInAnyOrderElementsOf(metricNames);
   }
 
+  @Test
+  void getMetricNamesFailed() {
+    try {
+      metadataService.getMetricNames("").block();
+    } catch (Exception ignored) {
+      final double value = meterRegistry.get("ceres.db.operation.errors")
```

Comment:
should this be in finally block? If test will pass if it does not raise exception.

Reply:
this counter will be only incremented in case of exception being raised as the counter increment logic is inside doOnError()

Reply:
Agree. As the test is guaranteed to raise exception, was wondering if the assertion should be in finally block. For any reason, if `metadataService.getMetricNames("").block()` does not raise the exception, this test will pass. It should fail ideally if method does not reach error flow.